src/status_im/data_store/chats.cljs

```diff
@@ -63,15 +72,46 @@
                                 :unviewedMessagesCount :unviewed-messages-count
                                 :unviewedMentionsCount :unviewed-mentions-count
                                 :lastMessage :last-message
-                                :active :is-active
                                 :lastClockValue :last-clock-value
                                 :invitationAdmin :invitation-admin
                                 :profile :profile-public-key})
+      rpc->type
+      (unmarshal-members members-reducer-cljs)
       (update :last-message #(when % (messages/<-rpc %)))
       (dissoc :members)))
 
-(fx/defn fetch-chats-rpc [cofx {:keys [on-success]}]
+(defn <-rpc-js [^js chat]
+  (-> {:name (.-name chat)
+       :description (.-description chat)
+       :color (.-color chat)
+       :active (.-active chat)
+       :timestamp (.-timestamp chat)
+       :alias (.-alias chat)
+       :identicon (.-identicon chat)
+       :muted (.-muted chat)
+       :joined (.-joined chat)
+
+       :chat-id (.-id chat)
+       :community-id (.-communityId chat)
+       :synced-from (.-syncedFrom chat)
+       :synced-to (.-syncedTo chat)
+       :deleted-at-clock-value (.-deletedAtClockValue chat)
+       :chat-type (.-chatType chat)
+       :unviewed-messages-count (.-unviewedMessagesCount chat)
+       :unviewed-mentions-count (.-unviewedMentionsCount chat)
+       :last-message (when (.-lastMessage chat)
+                       {:content {:text (.-lastMessage.text chat)}
+                        :content-type (.-lastMessage.contentType chat)})
+       :last-clock-value (.-lastClockValue chat)
+       :profile-public-key (.-profile chat)
+       :members (.-members chat)}
+      rpc->type
+      (unmarshal-members members-reducer-js)
```

Comment:
Part of this could be moved to status-go btw if that helps (not in this pr of course)